datastore/shared/services/environment_service.py

```diff
@@ -39,13 +40,10 @@ def is_dev_mode(self) -> bool:
         return value is not None and value.lower() in ("1", "on", "yes", "true")
 
     def get_from_file(self, name: str) -> str:
-        if name not in self.cache:
-            path = os.environ.get(name + "_FILE", None)
-            if path is not None:
-                with open(path) as file_:
-                    self.cache[name] = file_.read()
-            elif self.is_dev_mode():
-                self.cache[name] = os.environ.get(name, None)
-            else:
-                raise EnvironmentVariableMissing(name + "_FILE")
-        return cast(str, self.cache[name])
+        path = os.environ.get(name + "_FILE", None)
+        if self.is_dev_mode():
+            self.cache[name] = DEV_SECRET
+        elif path is not None:
```

Comment:
Looks really good! One "syntactic sugar":

```suggestion
        elif path := os.environ.get(name + "_FILE", None) is not None:
        # [...]
```

But this is not necessary

Reply:
As I wrote the ":=" and patch has issues. I won't add syntactic poison.

Reply:
Just for reference:
```suggestion
        elif (path := os.environ.get(name + "_FILE", None)) is not None:
```
should work since the `:=` operator takes a higher preference than `is`.